understanding/21/character-key-shortcuts.html

```diff
@@ -50,7 +50,7 @@ <h3>Disable Shortcuts</h3>
 
 <h3>Alternate Control</h3>
 
-<p>A keyboard-only user is in a long issues thread. While reading the thread she accidentally hits the S key, which moves focus to the search bar at the top of the document. This causes her to lose her place and her train of thought. However, a mechanism is provided to allow users to change character-key shortcuts. She changes the shortcut to include another key so she can avoid future interruptions.</p>
+<p>A keyboard-only user is in a long issues thread. While reading the thread they accidentally hit the S key, which moves focus to the search bar at the top of the document. This causes them to lose their place and her train of thought. However, a mechanism is provided to allow users to change character-key shortcuts. They change the shortcut to include another key so they can avoid future interruptions.</p>
```

Comment:
Still got "her" here. 

Second sentence should read "This causes them to lose their place and their train of thought."

Reply:
that's the exact change that this PR is proposing for this file, unless I'm missing something? Though @mbgower this was not changed to plural in your last round of "use plural rather than they/them" change of approach here, so I'll turn this into "Keyboard-only users..."

Reply:
```suggestion
<p>Keyboard-only users are in a long issues thread. While reading the thread they accidentally hit the S key, which moves focus to the search bar at the top of the document. This causes them to lose their place and train of thought. However, a mechanism is provided to allow users to change character-key shortcuts. They change the shortcut to include another key so they can avoid future interruptions.</p>
```

Reply:
changed to be plural here. and apologies @GrahamTheDevRel I managed to completely miss the second instance of "her" yet again while re-reading it...doh.

Reply:
> Though @mbgower this was not changed to plural in your last round of "use plural rather than they/them" change of approach here, so I'll turn this into "Keyboard-only users..."

I thought it was slightly more user specific, so left it as singular subject (and was only reviewing your proposed changes, so missed the second "her" too).  If you think it works as "Keyboard-only users", that's totally fine.